src/main/java/com/strandls/cca/service/impl/CCATemplateServiceImpl.java

```diff
@@ -262,6 +263,30 @@ public List<CCAField> getFilterableFields(HttpServletRequest request, String sho
 		return ccaFields;
 	}
 
+	@Override
+	public List<CCAField> getFilterableChartFields(HttpServletRequest request, String shortName, String language) {
+		if (shortName == null || "".equals(shortName))
```

Comment:
please dont directly compare with empty string. Use inbuilt java method which is much cleaner.

Reply:
ignoring this for now since its the same comparison is  used by other functions , we will refactor later.